docs/issues/house-rules.md

```
@@ -0,0 +1,329 @@
+
+No `visibleFor` — always shown for every character.
+
+### 13. New route: `src/routes/$characterId/settings.tsx`
+
+`createFileRoute("/$characterId/settings")` rendering `<HouseRulesSection />`.
+
+### 14. New components under `src/components/character/houseRules/`
+
+**`houseRulesSection.tsx`** — top-level section, analogous to `karmaSection.tsx`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)